test/integration/nested_model.cc

```diff
@@ -1371,3 +1371,87 @@ TEST(NestedReference, PlacementFrameElement)
     EXPECT_EQ(placementPose, pose.Inverse());
   }
 }
+
+/////////////////////////////////////////////////
+TEST(NestedModel, IncludeElements)
+{
+  const std::string modelRootPath = sdf::filesystem::append(
+      PROJECT_SOURCE_PATH, "test", "integration", "model");
+
+  sdf::setFindCallback(
+      [&](const std::string &_file)
+      {
+        return sdf::filesystem::append(modelRootPath, _file);
+      });
+
+  auto checkIncludeElem = [](sdf::ElementPtr _includeElem,
+                              const std::string &_uri, const std::string &_name)
+  {
+    ASSERT_NE(nullptr, _includeElem);
+    ASSERT_TRUE(_includeElem->HasElement("uri"));
+    EXPECT_EQ(_uri, _includeElem->Get<std::string>("uri"));
+
+    ASSERT_TRUE(_includeElem->HasElement("name"));
+    EXPECT_EQ(_name, _includeElem->Get<std::string>("name"));
+
+    ASSERT_TRUE(_includeElem->HasElement("pose"));
+    EXPECT_EQ(ignition::math::Pose3d(1, 2, 3, 0, 0, 0),
+        _includeElem->Get<ignition::math::Pose3d>("pose"));
+
+    ASSERT_TRUE(_includeElem->HasElement("placement_frame"));
+    EXPECT_EQ("link", _includeElem->Get<std::string>("placement_frame"));
+
+    ASSERT_TRUE(_includeElem->HasElement("plugin"));
+    auto pluginElem = _includeElem->GetElement("plugin");
+    ASSERT_TRUE(pluginElem->HasAttribute("name"));
+    EXPECT_EQ("test_plugin", pluginElem->Get<std::string>("name"));
+    ASSERT_TRUE(pluginElem->HasAttribute("filename"));
+    EXPECT_EQ("test_plugin_file", pluginElem->Get<std::string>("filename"));
+  };
+
+  // Test with //world/include
+  {
+    std::ostringstream stream;
+    stream << R"(
+    <sdf version='1.8'>
```

Comment:
nit (minor) Inconsistent usage of quotes, `'` and `"`

Reply:
I'm terrible at staying consistent with this. Changed to `'`